apps/site/assets/css/_trip-plan-widget.scss

```diff
@@ -104,9 +116,138 @@
   justify-content: flex-start;
 }
 
-.c-trip-plan-widget__reverse-control {
+.c-trip-plan-widget__reverse-control-vert {
   color: $brand-primary;
   cursor: pointer;
   margin-left: auto;
   user-select: none;
 }
+
+.m-mode-hub__guides-wrapper {
```

Comment:
is this class present for all the other contexts which the widget is being used?

Reply:
Was only present in index.html. Updated to col-xs-12 which is present on all contexts besides the homepage.